src/main/java/com/google/sps/servlets/TripServlet.java

```diff
@@ -179,44 +219,30 @@ private void eventDoGet(HttpServletResponse response) throws IOException {
   }
 
   /**
-   * Make the servlet cleaner.
-   * Searches through the parameters and creates the events and puts them into
-   * datastore.
+   * Creates the events and puts them into datastore.
+   * @param date Trip date
+   * @param pois List<String> of pois
+   * @param travelTimes List<Integer> of travel times in minutes
    */
-  private void eventDoPost(HttpServletRequest request, HttpServletResponse response) 
+  private void eventDoPost(String date, List<String> pois, List<Integer> travelTimes) 
       throws IOException { 
     
-    // Print out params to site to verify retrieval of "start trip" user input.
-    Enumeration<String> params = request.getParameterNames();
-
-    // get date of trip
-    String date = request.getParameter("inputDayOfTravel");
-
     // set startDateTime
     LocalDateTime startDateTime = LocalDateTime.of(LocalDate.parse(date), LocalTime.of(10, 0));
 
-    // search through all the parameters looking for pois
-    while (params.hasMoreElements()) {
-      String p = params.nextElement();
-
-      /** 
-       * for each poi create the necessary fields, this will change
-       * as we are able to pull from the maps backend api
-       */
-      if (p.contains("poi")) {
-        String address = request.getParameter(p);
-        String name = address.split(",")[0];
-        Event event = new Event(name, address, startDateTime, HALF_HOUR);
-        Entity eventEntity = event.eventToEntity();
-
-        // put entity in datastore
-        DatastoreService datastore = 
-                                DatastoreServiceFactory.getDatastoreService();      
-        datastore.put(eventEntity);
-
-        // sets start time for next event 2 hours after start of prev
-        startDateTime = startDateTime.plusMinutes(Long.valueOf(NINETY_MINS));
-      }
+    int travelTimeIndex = 1;
```

Comment:
Yes you're right! Initially I was going with the assumption that the user reaches the first POI at 10 am, but I realize that it makes more sense to assume that the user would leave the hotel at 10 am, so I updated the start time of the first POI to 10am + travelTimeIndex[0] like you said since the first travelTimeIndex[0] represents the time from the hotel to the first POI. I added a comment to explain this in the code.